test/e2e/metriconly/e2e_npd_test.go

```diff
@@ -49,7 +49,7 @@ var boskosProjectType = flag.String("boskos-project-type", "gce-project",
 	"specifies which project type to select from Boskos.")
 var boskosServerURL = flag.String("boskos-server-url", "http://boskos.test-pods.svc.cluster.local",
 	"specifies Boskos server URL.")
-var boskosWaitDuration = flag.Duration("boskos-wait-duration", 5*time.Minute,
+var boskosWaitDuration = flag.Duration("boskos-wait-duration", 2*time.Minute,
```

Comment:
Does this value need to be updated when we add more tests (seems so from the commit description?)

Reply:
No. When we add more tests, if they cannot finish in 10 minutes, we will find a way to better scale them. I have a few ideas, and when that day is close, we can implement it:

1. We can increase the parallelism. If the test is taking too much time to finish.
All the tests run in parallel. I set the parallelism to 3 for now. We can change it easily.

2. We can rent 1 Boskos project, and ask all Ginkgo nodes to use it. If the Boskos renting is slow/flaky.

3. We can increase the test timeout.
It's currently set to 10 minutes. This is pretty ambitious for e2e test. We might need to increase it.


Right now, we are not really face any scaling issue. So I didn't prioritize implementing any of the above ideas :)
When that day comes close, we can do it. They are not really hard to do.

Reply:
Cool. Maybe document those ideas in an issue.

Reply:
Sounds good :) documented at #398 398